src/Web.cpp

```diff
@@ -457,6 +457,7 @@ void webserverStart(void) {
 
 		// RFID
 		wServer.on("/rfid", HTTP_GET, handleGetRFIDRequest);
+		wServer.addRewrite(new OneParamRewrite("/rfid/details", "/rfid?details=true"));
```

Comment:
OneParamRewrite is usually for path parameters in the form of /path/{some_param} -> /path?some_param={some_param} because AsyncWebServer cannot handle path params natively. 

Does this even work and do something? Even if it does, using the "normal" AsyncWebRewrite makes more sense: https://github.com/me-no-dev/ESPAsyncWebServer#param-rewrite-with-matching

Reply:
Yes it works fine, do you have a concrete improved code here?

Reply:
Didn't test it, but just replacing 'OneParamRewrite' with 'AsyncWebRewrite' should do the same thing.